### PR TITLE
Remove another Telerik reference

### PR DESCRIPTION
**Pull request by jeremy-farrance**
Provides a very slight performance improvement. :)

### CODE REVIEW

File: content/getting-started/setup/telerik-removal/index.md [hunk 1]
```diff
@@ -71,7 +71,10 @@ Navigate to `Settings > Extensions (Libraries)` in the `Persona Bar` and uninsta
 Navigate to `Settings > Extensions (Libraries)` in the `Persona Bar` and uninstall the `DotNetNuke Deprecated Website Codebehind files` extension.  Be sure to check the `Delete Files` checkbox.
 
 ### Step 12
-Open the `web.config` file within the site root and search for "Telerik".  Delete any lines that reference it.
-
+Open the `SiteUrls.config` file within the site root and search for "Telerik".  Delete all <RewriteRule>...</RewriteRule> that reference it.
```
Comment (david-poindexter) [inline content/getting-started/setup/telerik-removal/index.md:74]:
```suggestion
Open the `SiteUrls.config` file within the site root and search for "Telerik".  Delete all rewrite rules, `<RewriteRule>...</RewriteRule>`, that reference it.
```

**Review by david-poindexter — APPROVED** [review]
Looks good @jeremy-farrance - thanks! 🎉 
By the way, this makes me wonder why the Config Manager doesn't allow for editing `SiteUrls.config`.  🤔 It probably should, right?

**Comment by jeremy-farrance** [comment]
Psst, `SiteUrls.config` is in there already (Config Manager dropdown).

**Comment by david-poindexter** [comment]
Hmm, I don't see it...

image

**Comment by jeremy-farrance** [comment]
> Hmm, I don't see it...

I just tried it on 2 different v9.01.01 sites. It appears to be showing all *.config files in the root. I was NOT aware that it did that. Did it always work that way? Or did Will change it when he added Robots.txt recently? I just added the Test-McTest one in the root, restarted the app and...
 
image

**Comment by david-poindexter** [comment]
Okay, we are both kinda right.  😆 

A clean install of DNN 9.9.1 does not have a `SiteUrls.config` file.  😉 

`*.config` files should be available for editing within the `Config Manager`, as well as `Robots.txt`.

**Comment by david-poindexter** [comment]
Wait a minute - I think I am wrong.  It does exist in `~/Config`.  So...not sure why it is not showing up in `Config Manager`.  Hopefully, someone can look into this or shed some light on the situation.

**Comment by bdukes** [comment]
The Config Manager only edits files in the site root, not the `~/Config` directory. The files in `~/Config` are the defaults, i.e. they're included in upgrade packages so that edited config files aren't overridden.

**Pull request merged**